Automattic/wp-calypso
# Revert "Plans section: Modernized header and navbar / part 1"

Reverts Automattic/wp-calypso#72960

The above change breaks the header of a domain only view. So the change is to be reverted.

![image](https://user-images.githubusercontent.com/3422709/217629514-28209710-6abe-4e27-a9ae-44c610080376.png)

Context :  p1675875224401949-slack-C02FMH4G8

@chriskmnds please do have a look at resolving this issue and merging this change back :) .


### Steps to test
1 - Create a free site without a domain
2 - Click on the following card

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/3422709/217630428-2600644d-db26-4b84-9f57-bba7b8f34d7f.png">

3 - Make sure the following unbroken header view is visible.

![image](https://user-images.githubusercontent.com/3422709/217630504-60755e10-6a6c-4eda-9605-462518fe872b.png)
